app/controllers/admin/invitations_controller.rb

```diff
@@ -2,11 +2,11 @@ class Admin::InvitationsController < Admin::ApplicationController
   include Admin::WorkshopConcerns
 
   def update
-    set_and_decorate_workshop
+    @invitation_manager = Managers::InvitationManager.new(params[:workshop_id], invitation_id, current_user)
+    @workshop = WorkshopPresenter.decorate(@invitation_manager.workshop)
     authorize @workshop
-    set_invitation
-
-    message = update_attendance(attending: params[:attending], attended: params[:attended])
+    @invitation = @invitation_manager.invitation
+    message = @invitation_manager.update_attendance(attending: params[:attending], attended: params[:attended])
```

Comment:
In order to _know_ that this argument is of truthy or falsy, Rails offers [a helpful cast method](https://api.rubyonrails.org/classes/ActiveModel/Type/Boolean.html):

```suggestion
    message = @invitation_manager.update_attendance(attending: ActiveRecord::Type::Boolean.new.cast(params[:attending]), attended: params[:attended])
```

Here I tried a few values in a console:

```
DEV (main)> ActiveRecord::Type::Boolean.new.cast("TRUE")
true
DEV (main)> ActiveRecord::Type::Boolean.new.cast("T")
true
DEV (main)> ActiveRecord::Type::Boolean.new.cast("")
nil
DEV (main)> ActiveRecord::Type::Boolean.new.cast(false)
false
DEV (main)> ActiveRecord::Type::Boolean.new.cast("f")
false
DEV (main)> ActiveRecord::Type::Boolean.new.cast("false")
false
```

Quoting the rules:

- "false", "f", "0", 0 or any other value in [FALSE_VALUES](https://api.rubyonrails.org/classes/ActiveModel/Type/Boolean.html#FALSE_VALUES) will be coerced to false.
- Empty strings are coerced to nil.
- All other values will be coerced to true.

Hope this helps!

Reply:
Done!